dts/bindings/clock/st,stm32f4-pll-clock.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32F4 PLL
```

Comment:
It would be nice to document PLL "formulas", i.e. output frequency = f(input frequency, properties)

Reply:
Right, this will be clearer! Adding this.